backend/controllers/appointmentConfirmDeletion.js

```diff
@@ -0,0 +1,86 @@
+const packAppointmentDetails = (req, details) => {
+  req.flash('appointmentDetails', details)
+}
+
+const unpackAppointmentDetails = req => {
+  const appointmentDetails = req.flash('appointmentDetails')
+  if (!appointmentDetails || !appointmentDetails.length) throw new Error('Appointment details are missing')
```

Comment:
This would result in HTTP 500 error being posted to dps_alerts. I'm not sure if this is the same but we've had situations previously where a user has hit F5 and triggered an error. Would it be better to redirect back to the start of the journey?

Reply:
That was NN-3204 - actually we never understood why that was happening, just that it did. But thinking about it, yes, redirecting is probably better - it would have to be the `view-all-appointments` page though